modules/core_project_factory/main.tf

```diff
@@ -21,6 +21,17 @@ resource "random_id" "random_project_id_suffix" {
   byte_length = 2
 }
 
+resource "random_string" "random_project_id_suffix" {
+  count   = var.random_project_id_string_method ? 1 : 0
+  length  = var.random_project_id_length
+  special = false
+  upper   = false
+
+  lifecycle {
+    ignore_changes = all
+  }
```

Comment:
Do we need to ignore changes during normal usecase or is this for the import workflow? Would be good to add a comment.

Reply:
This was for import, but as that is no longer a relevant workflow I will remove it.